spec.bs

```diff
@@ -248,6 +255,92 @@ If using a centralized mechanism for fetching keys, user agents should have a re
 
 When using a registration process, it is recommended that user agents apply an expiration date to registration requests, to allow for the removal of deprecated or no longer active issuers.
 
+VOPRF Methods {#voprf-methods}
+====================================
+
+This document encodes protocol messages in the TLS presentation language from section 3 of [[!RFC8446]].
+
+For <dfn>Issue</dfn> (corresponding to [=BlindEvaluate=] function of the [[!VOPRF]] protocol), the <dfn>IssueRequest</dfn> and <dfn>IssueResponse</dfn> are serialized as:
+
+```
+// Scalars are elliptic curve scalars of length Ns (determined by the curve, 48 for P-384).
+// ECPoints are elliptic curve points encoded using the X9.62 Uncompressed point representation (determined by the curve, 97 for P-384).
+
+struct {
+  uint16 count;
+  ECPoint nonces[count];
+} IssueRequest;
+
+struct {
+  ECPoint blinded;
+  ECPoint evaluated;
+} SignedNonce;
+
+struct {
+  Scalar c;
+  Scalar s;
+} DLEQProof;
+
+struct {
+  uint16 issued;
+  uint32 key_id = keyID;
```

Comment:
This looks like the constant syntax from https://www.rfc-editor.org/rfc/rfc8446#section-3.7, but `keyID` doesn't look like a constant.